ui/src/app/job-details/panels/panels.component.ts

```diff
@@ -45,6 +50,8 @@ export class JobPanelsComponent implements OnChanges {
     this.inputs = Object.keys(this.job.inputs || {}).sort();
     this.outputs = Object.keys(this.job.outputs || {}).sort();
     this.labels = Object.keys(this.job.labels || {}).sort();
+    this.extensions = Object.keys(this.job.extensions || {}).sort();
```

Comment:
Perhaps it would make sense to do filtering by type here in addition to or instead of the blacklisting? Wouldn't `tasks` get rendered with the current code?

I would also not be opposed to a whitelst approach, or alternatively just hardcoding extension fields into the template; not sure if we plan to give any of these special treatment.

Reply:
Good point, I think whitelist is a better approach so that any new extensions aren't automatically displayed (in case they are the wrong type). Changed to that.